in_toto/in_toto_record.py

```diff
@@ -68,6 +68,8 @@
                         for additional info.
   --base-path <path>    Record 'materials/products' relative to <path>. If not
                         set, current working directory is used as base path.
+  --lstrip-paths <path> If a prefix path is passed, the prefix is left stripped
+                        from the path of every artifact that contains the prefix.
```

Comment:
This is not what the actual help message says (see comment above). They should match.